src/components/Link/Link.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withTranslation } from 'react-i18next';
+import { compose } from 'lodash/fp';
+import { ReactComponent as LinkIcon } from './Link.svg';
+
+import './Link.scss';
+
+const propTypes = {
+  href: PropTypes.string.isRequired,
+  children: PropTypes.node,
+};
+
+const defaultProps = {
+  children: null,
+};
+
+const Link = ({ href, children }) => {
```

Comment:
I've added this new component on purpose as we will have several topics with popup links.